client/lib/main.dart

```diff
@@ -47,10 +46,6 @@ class MyApp extends StatelessWidget {
           },
         ),
         ChangeNotifierProvider(create: (_) => CourseState()),
-        ChangeNotifierProxyProvider<User, RecentsState>(

```

Comment:
not sure why this was removed in my commits, is this the intended structure?

Reply:
i removed this, its not actually being used and was leftover

Reply:
got it